Python-Function-Complex.md

```diff
@@ -0,0 +1,54 @@
+# Python Complex()
+
+`complex()` is a built-in function in Python 3 that hanldles the creation of a complex number according to the arguments passed. It returns a complex number with the value `real + imag*1j` when called in form of `complex(real, imag)` or converts a string or number to a complex number.
+
+## Arguments and Return Values
+
+Two types of arguments can be passed.
```

Comment:
Don't leave the sentence just like that. Add a line stating you are going to discuss those two arguments below.